### DISCUSSION TITLE
exclude-severity in workflows

### DISCUSSION THREAD

**Discussion by savushkin-yauheni** [projectdiscovery/nuclei#2553] [Q&A] [answered]
Hi all.

Does `exclude-severity` option work in workflows ? It seems that it's not. 

E.x. workflow:

```
id: springboot-workflow

info:
  name: Spring Boot Security Checks
  author: dwisiswant0
  description: A simple workflow that runs all Spring Boot related nuclei templates on a given target.

workflows:
  - tags: springboot
```

run nuclei:

```
nuclei -duc  -target https://google.com -w template.yaml  -c 30   -timeout 30   -es info    -etags jolokia   -json --debug-req
```

You will see requests from `technologies/springboot-actuator.yaml` templates which is `info`. And some other `info` templates.

Am I doing something wrong ? 

Thanks for your time!

**Comment by ehsandeep** [comment, marked as answer]
@savushkin-yauheni it is not supported - https://github.com/projectdiscovery/nuclei/discussions/1894
